modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -111,6 +112,29 @@ def diff(rep_data)
         end
       end
 
+      def self.get_suffix(first_name, last_name)
+        first_suffix = first_name.split.last if first_name.split.count > 1
+        last_suffix = last_name.split.last if last_name.split.count > 1
+        [first_suffix, last_suffix]
+      end
+
+      def self.get_representatives(first_name, last_name)
+        suffix = get_suffix(first_name, last_name)
+
+        representatives = where('lower(first_name) = ? AND lower(last_name) = ?', first_name&.downcase,
+                                last_name&.downcase)
+
+        if representatives.blank? && suffix.present?
+          # check without suffix
+          first_name = first_name.delete(suffix[0]).strip if suffix[0].present?
+          last_name = last_name.delete(suffix[1]).strip if suffix[1].present?
+
```

Comment:
@tycol7 What do you think about the first name "sufffix" or potential prefix?

Reply:
I like how we have it now, in case the first name has the suffix. I can imagine a scenario where the nightly scrape stores the suffix in the first name, e.g., `Last, First Jr.`.

If prefixes become a problem, I'm sure we can solve it in a future task.